### ISSUE TITLE
Better line colour choice 

### ISSUE THREAD

**Issue by joncison** [IFB-ElixirFr/edam-browser#21]
To help colour-perceptive-impaired people (well me at least) a bit :) 

I struggle to see the difference in colour used for Operation and Topic here:
![capture](https://user-images.githubusercontent.com/1506863/40416485-4cee53ae-5e75-11e8-8db2-434cca6c404c.PNG)

I recall we had a sort-of "formal" colour schema for EDAM as used on Kenzo's EDAM poster, right @hmenager ?

**Comment by joncison** [comment]
... or if we are already using the formal colour scheme - just close this issue and I'll concentrate harder when looking.

**Comment by bryan-brancotte** [comment]
Hi @joncison 
Surprisingly the color used and then rendered are not the same ! I guess that it might me a transparency issue or something on d3 side, I'll take a closer look.

**Comment by joncison** [comment]
Yup!  The colours in https://github.com/edamontology/edamontology/issues/340#issuecomment-363754915 look good!

**Comment by raashika03** [comment]
@bryan-brancotte any thing left in this to be covered or can be closed?
